### PR TITLE
Add code documentation

### LINKED ISSUE #191: Add code documentation

### PR DESCRIPTION
**Pull request by fsoupimenta**
Closes #191

### CODE REVIEW

**Review by LBeghini — APPROVED** [review]

File: source/controller/wizard_controller.py [hunk 1]
```diff
@@ -166,6 +166,14 @@ def verify_and_save_project_folder(self):
             self.update_complete_project_files_page(complete_project_location=False)
 
     def save_project_description(self):
+        """Takes a 'str' added by the user in the wizard 'project_description_input',
+         save in the 'wizard_information_store' and set the 'project_description' in the review page.
```
Comment (LBeghini) [inline source/controller/wizard_controller.py:170]: This method does not take a string, as none is passed in the parameter.  It is not a good practice to mention the internal variables the method uses, so no 'wizard_information_store', for example. 

```suggestion
        """Updates the project description throughout the places it is being used
```
Maybe it is even a good idea to change the method name to `update_project_description`, given what it does.

File: source/controller/wizard_controller.py [hunk 2]
```diff
@@ -166,6 +166,14 @@ def verify_and_save_project_folder(self):
...
+
+        Parameters
+        -----------
+        description : str
+            It is an input string added in the 'project_description_input'
```
Comment (LBeghini) [inline source/controller/wizard_controller.py:175]: It does not receive any parameters.
```suggestion

```

File: source/controller/wizard_controller.py [hunk 3]
```diff
@@ -166,6 +166,14 @@ def verify_and_save_project_folder(self):
             self.update_complete_project_files_page(complete_project_location=False)
 
     def save_project_description(self):
```
Comment (LBeghini) [inline source/controller/wizard_controller.py:168]: Do not forget to add returns and types to the methods:
```suggestion
    def update_project_description(self) -> None:
```

**Review by LBeghini — CHANGES_REQUESTED** [review]

File: source/controller/wizard_controller.py (hunk 3 above)
Comment (rosaigor138) [reply source/controller/wizard_controller.py:168]: Ok, thank you for the help. I will change my errors

**Pull request closed without merging**